src/components/themed/TransactionListRow.tsx

```diff
@@ -163,36 +163,42 @@ export function TransactionListRow(props: Props) {
   })
 
   const handleLongPress = useHandler(() => {
-    triggerHaptic('impactLight')
     const url = sprintf(currencyInfo.transactionExplorer, transaction.txid)
     const shareOptions = {
       url
     }
     Share.open(shareOptions).catch(e => showError(e))
   })
 
-  return (
-    <ClickableRow paddingRem={[0, 1]} onPress={handlePress} onLongPress={handleLongPress}>
-      <View style={styles.iconContainer}>
-        <View style={[styles.iconArrowsContainer, transactionStyle, thumbnailPath ? null : styles.iconArrowsContainerBackground]}>
-          {thumbnailPath ? null : transactionIcon}
-        </View>
-        <FastImage style={styles.icon} source={{ uri: thumbnailPath }} />
+  const icon = (
+    <View style={styles.iconContainer}>
+      <View style={[styles.iconArrowsContainer, transactionStyle, thumbnailPath ? null : styles.iconArrowsContainerBackground]}>
+        {thumbnailPath ? null : transactionIcon}
       </View>
-      <View style={styles.transactionContainer}>
-        <View style={styles.transactionRow}>
-          <EdgeText style={styles.transactionText}>{transactionText}</EdgeText>
-          <EdgeText style={isSentTransaction ? styles.negativeCryptoAmount : styles.positiveCryptoAmount}>{cryptoAmountString}</EdgeText>
-        </View>
-        <View style={styles.transactionRow}>
-          <View style={styles.categoryAndTimeContainer}>
-            {categoryText && <EdgeText style={styles.category}>{categoryText}</EdgeText>}
-            <EdgeText style={pendingStyle}>{pendingText}</EdgeText>
+      <FastImage style={styles.icon} source={{ uri: thumbnailPath }} />
+    </View>
+  )
+
+  return (
+    <CardUi4 icon={icon} onPress={handlePress} onLongPress={handleLongPress}>
+      <SectionView>
```

Comment:
Doesn't the card already include the `SectionView` functionality inside itself? This seems redundant.

Reply:
You're right, I think I previously had some special handling that tweaked things but since removed it.